internal/transport/transport_test.go

```diff
@@ -462,6 +462,8 @@ func setUpWithNoPingServer(t *testing.T, copts ConnectOptions, done chan net.Con
 // TestInflightStreamClosing ensures that closing in-flight stream
 // sends status error to concurrent stream reader.
 func TestInflightStreamClosing(t *testing.T) {
+	t.Parallel()
```

Comment:
I believe @dfawley has some comments about running parallel tests.

Reply:
I did see a considerable reduction in test execution time with parallel tests :)